src/vm/gdbjit.cpp

```diff
@@ -1637,21 +2135,23 @@ bool NotifyGdb::BuildSectionTable(MemBuf& buf)
             pSh->sh_entsize = 0;
     }
 
-    buf.MemPtr = reinterpret_cast<char*>(sectionHeaders);
+    sectionHeaders.SuppressRelease();
+    buf.MemPtr = reinterpret_cast<char*>(sectionHeaders.GetValue());
     buf.MemSize = sizeof(Elf_Shdr) * (SectionNamesCount - 1);
     return true;
 }
 
 /* Build the ELF header */
 bool NotifyGdb::BuildELFHeader(MemBuf& buf)
 {
-    Elf_Ehdr* header = new (nothrow) Elf_Ehdr;
-    buf.MemPtr = reinterpret_cast<char*>(header);
+    NewHolder<Elf_Ehdr> header = new (nothrow) Elf_Ehdr;
```

Comment:
The holder here is not necessary - you always call SuppressRelease(), so it is better to use just a plain pointer.